portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase

```diff
@@ -57,12 +57,55 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8148 | Test Summary: Individuals usage update in real time"
-	@ignore = "true"
 	@priority = "5"
 	test IndividualsUpdatesInRealTime {
+		ACUtils.addSiteAndPage();
+
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "${propertyName}");
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUsage();
+
+		var currentIndividuals = selenium.getText("ACUsage#USAGE_CURRENT_INDIVIDUALS");
+
+		ACUtils.launchDXP();
 
-		// TODO LRAC-8148 Usage#IndividualsUpdatesInRealTime pending implementation
+		JSONUser.addUser(
+			userEmailAddress = "userea@liferay.com",
+			userFirstName = "userfn",
+			userLastName = "userln",
+			userScreenName = "usersn");
 
+		ACDXPSettings.syncNewContactData();
+
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "userea@liferay.com",
+			userLoginFullName = "userfn userln");
+
+		Navigator.gotoSitePage(
+			pageName = "AC Page",
+			siteName = "Site Name");
+
+		ACUtils.closeAllSessions();
+
+		ACUtils.launchAC();
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUsage();
+
+		var newIndividuals = selenium.getText("ACUsage#USAGE_CURRENT_INDIVIDUALS");
+
+		if ("${currentIndividuals}" != "${newIndividuals}") {
+			AssertTextNotEquals(
+				locator1 = "ACUsage#USAGE_CURRENT_INDIVIDUALS",
+				value1 = "${currentIndividuals}");
+		}
```

Comment:
Hi @AfonsoHSantos, I think this `if` is flawed, because if the values are equal, the test will pass and the Assert will not be done, this way the test will not show the possible bug in this number update. What do you think?

Reply:
@AfonsoHSantos also, instead of asserting that currentIndividuals != newIndividuals, it would be better to assert that newIndividuals is +1 from the original value.  That way we can be sure the individual count is only going up by one when one new individual is added.

Reply:
Hi @christianstokes! good idea, I applied it.